packages/react/playground/next-latest/src/app/components/Teaser.tsx

```diff
@@ -1,9 +1,6 @@
 import { storyblokEditable } from '@storyblok/react/rsc';
-import { headers } from 'next/headers';
```

Comment:
@maoberlehner note for the review:

I'm unsure about what was the intention of this here. It made the playground fail (both Next 15 and 16) as soon as you wanted to use the bridge for live editing, as for that you need the `StoryblokProvider` pattern, and that makes the component a client component.

I also realized that the playground didn't include StoryblokProvider, and bridge wasn't enabled. I updated to include that, according to how it's documented https://www.storyblok.com/docs/guides/nextjs